app/services/pricing/service.go

```diff
@@ -324,9 +335,17 @@ func (s *Service) updatePriceInfoContainers(nativeAsset *asset.NativeAsset, toke
 			continue
 		}
 		wrappedAssetInfo, _ := s.assetsService.FungibleAssetInfo(networkId, wrappedToken)
+		
+		defaultMinAmount := tokenPriceInfo.DefaultMinAmount
```

Comment:
Is it correct to get defaultMinAmount from tokenPriceInfo.DefaultMinAmount?
More likely not because decimals may be different, since this function updates the amount for wrapped tokens on other networks?

Maybe it is better to have defaultUsdPrice instead of defaultMinAmount, so when we implement the custom fees
the calculations will work relatively good for the special users also.